modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/dnd_table/Cell.tsx

```diff
@@ -127,7 +118,11 @@ const Cell = ({
 
 	return (
 		<div
-			className={classNames(heading ? 'dnd-th' : 'dnd-td', className)}
+			className={classNames(
+				heading ? 'dnd-th' : 'dnd-td',
+				`cell-${columnName}`,
```

Comment:
We will need to do some transformation on the column name because already some of the commerce column names have characters that are not allowed such as a comma (`,`). You can see this if you navigate to commerce products in the admin:

![Screenshot 2024-09-17 at 12 02 20 PM](https://github.com/user-attachments/assets/e4f0a4d1-0c4c-4395-a199-0a83b11a3f26)

Reply:
Additionally it might be nice to make this a util so that the cellClassNames can be easily applied to the cells in ClayTable to:
1. https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/ClayTable.tsx#L189
2. https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/ClayTable.tsx#L224
3. https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/ClayTable.tsx#L287
4. https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/ClayTable.tsx#L350
5. https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/ClayTable.tsx#L393

Given that we are close to switching to the ClayTable, it would be great if this fix was applied there as well so we don't have to fix this again later.

Reply:
> We will need to do some transformation on the column name because already some of the commerce column names have characters that are not allowed such as a comma (,). You can see this if you navigate to commerce products in the admin:

We can do a simple quick and dirty fix: cut `,LANG` and convert other non-alphabet characters into dashes. Just to have in mind, auto-generation is not suppose to be 100% perfect. That's why we should plan [Allow setting of custom FDS table cell CSS classes
](https://liferay.atlassian.net/browse/LPD-10394).

Reply:
I've created a utility to do the transformation for the cell classes and apply to `ClayTable`. Not sure if it is worth as the latter is a real table and will fit the columns better.

Reply:
That makes sense @ethib137 we'll do so that we don't divert styling API across table implementations. Thanks for pointing it out